### PR TITLE
Api 12433/handle oauth2 security type

### PR DESCRIPTION
**Pull request by peyton-lucas**
Adding logic to allow promptForSecurityValues function to handle the oauth2 security scheme type. 

Handles security scheme of type:oauth2 (see Security Scheme Object in OAS spec https://swagger.io/specification/) in order to handle some of the API's such as Clinical Health for LOAST.

### CODE REVIEW

File: src/commands/positive.ts [hunk 1]
```diff
@@ -196,41 +196,60 @@ export default class Positive extends Command {
...
-      const bearerSecurityName = securityTypes[OASSecurityType.HTTP].key;
-      let value;
+      });
+    // Wrap if statements in loop to iterate over securities array to check for apiKey or bearer_token(s)
+    let apiKey = flags.apiKey;
+    let token = flags.bearerToken;
+    for (const security of securities) {
+      if (security.type === OASSecurityType.APIKEY) {
+        const apiSecurityName = security.key;
+        const value =
```
Comment (mheusgit) [inline src/commands/positive.ts:214]: While the name "value" isn't something new it doesn't give much insight.

Comment (mheusgit) [reply src/commands/positive.ts:214]: Same for 233

**Review by mheusgit — COMMENTED** [review]
Besides the small variable naming item it looks good to me.

**Comment by peyton-lucas** [comment]
Ok cool. I'll get that variable renamed today, so we can move this forward.

**Comment by peyton-lucas** [comment]
Updated variable names.

**Comment by mheusgit** [comment]
Was about to approve but we have a conflict...

**Review by mheusgit — DISMISSED** [review]
Small fix is in place

**Review by mheusgit — APPROVED** [review]
Change approved

**Pull request merged**